Automattic/wp-calypso
# Migrate SEO preview pane styles

#### Changes proposed in this Pull Request

Migrate styles used in SEO pane inside preview when previewing "Search & Social."

#### Testing instructions

1. Select a post and click “Preview”
2. Select “Search & Social” from the dropdown in the upper left of the preview pane.
3. Check that SEO preview pane is styled correctly

##### Unstyled
<img width="1652" alt="screen shot 2018-10-03 at 12 04 32 am" src="https://user-images.githubusercontent.com/10561050/46371576-4397b580-c6bb-11e8-913f-1720f9745498.png">

##### Styled
<img width="796" alt="screen shot 2018-10-03 at 12 04 10 am" src="https://user-images.githubusercontent.com/10561050/46371590-4beff080-c6bb-11e8-9ba0-eecee42d6a22.png">

#27515
